index.html

```diff
@@ -1,111 +1,579 @@
+          </li>
+          <li>
+            <a href="Blog.html" class="nav-link">Blog</a>
+          </li>
+          <li>
+            <a href="Catalog.html" class="nav-link">Catalog</a>
+          </li>
+          <li>
+            <a href="Contact-us.html" class="nav-link">Contact Us</a>
+          </li>
+        </ul>
+      </nav>
+      <div>
+        <input type="text" size="2" />
+        <button><a href="link" class="button">Discard</a></button>
+        <button><a href="link" class="button button-blue">Connect</a></button>
+      </div>
+    </header>
+    <main>
+      <section class="statistic">
+        <ul>
+          WITH BINABOX EXPLORE NFT COLLECTION
+          <li>We are tne best way to check the rarity on NTF collection</li>
+          <li>Total item<span class="bold-text">2240+</span></li>
+          <li>Profiles whitelisted<span class="bold-text">1000+</span></li>
+        </ul>
+        <button type="submit">Connect walet</button>
+        <button type="submit">whitelist now</button>
+      </section>
+      <section class="catalog">
+        <div>
+          <h2><span class="folder-icon"></span>Catalog</h2>
+          <div></div>
+          <a href="catalog.html" class="link">See all</a>
+          <button>&#8249;</button>
+          <button>&#8250;</button>
+        </div>
+        <div>
+          <div class="catalog-info-item">
+            <img src="url" alt="text" />
+            <span class="lable">NEW</span>
+            <ul>
+              <li><span class="bold-text">$199</span></li>
+              <li>Name Product</li>
+            </ul>
+          </div>
+          <div class="catalog-info-item">
+            <img src="url" alt="text" />
+            <span class="lable">NEW</span>
+            <ul>
+              <li><span class="bold-text">$199</span></li>
+              <li>Name Product</li>
+            </ul>
+          </div>
+          <div class="catalog-info-item">
+            <img src="url" alt="text" />
+            <span class="lable">NEW</span>
+            <ul>
+              <li><span class="bold-text">$199</span></li>
+              <li>Name Product</li>
+            </ul>
+          </div>
+          <div class="catalog-info-item">
+            <img src="url" alt="text" />
+            <span class="lable">NEW</span>
+            <ul>
+              <li><span class="bold-text">$199</span></li>
+              <li>Name Product</li>
+            </ul>
+          </div>
+          <div class="catalog-info-item">
+            <img src="url" alt="text" />
+            <span class="lable">NEW</span>
+            <ul>
+              <li><span class="bold-text">$199</span></li>
+              <li>Name Product</li>
+            </ul>
+          </div>
+        </div>
+      </section>
+      <section class="blog">
+        <h2>Our blog</h2>
+        <article class="post">
+          <img src="#" alt="text" />
+          <span class="lable">NTF</span>
+          <span class="folder icon">Tony Nguyen</span>
```

Comment:
This and the next element should be combined into one block according to the design. Also pay attention to the class name. These names don't match the content of this block

Reply:
The article should have two blocks. A block with an image and label, a block with a description. Did I understand corectly?

Reply:
These elements should be combined into one block. You can use 'div' tag for it

<img width="443" alt="Screenshot 2024-11-06 at 6 13 27 PM" src="https://github.com/user-attachments/assets/2510d458-3850-447f-beac-ed087eaded4f">

Reply:
done

Reply:
done